Fails to build with with `-Werror=implicit-function-declaration`

Quoting from the Debian bug report [1]:

kcemu FTBFS with -Werror=implicit-function-declaration which was
recently enabled by default in dpkg 1.22.6 to support the 64-bit time_t
transition. For more information, see
https://wiki.debian.org/qa.debian.org/FTBFS#A2024-03-13_-Werror.3Dimplicit-function-declaration

```
gcc -DHAVE_CONFIG_H -I. -I../../../include/kc  -I../../../include -Wdate-time -D_FORTIFY_SOURCE=2  -g -O2 -Werror=implicit-function-declaration -ffile-prefix-map=/build/kcemu-0.5.2+dfsg=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection -c -o sys_socket_create.o sys_socket_create.c
sys_socket_create.c: In function 'sys_socket_create':
sys_socket_create.c:44:9: error: implicit declaration of function 'perror' [-Werror=implicit-function-declaration]
   44 |         perror("setsockopt() failed");
      |         ^~~~~~
cc1: some warnings being treated as errors
make[4]: *** [Makefile:518: sys_socket_create.o] Error 1
```

--- a/src/sys/mingw/sys_socket_create.c
+++ b/src/sys/mingw/sys_socket_create.c
@@ -17,6 +17,7 @@
  *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+#include <stdio.h>
 #include <winsock2.h>
 
 #include "kc/config.h"

--- a/src/sys/unix/sys_socket_create.c
+++ b/src/sys/unix/sys_socket_create.c
@@ -18,6 +18,7 @@
  */
 
 #include <fcntl.h>
+#include <stdio.h>
 #include <unistd.h>
 #include <sys/types.h>
 #include <sys/socket.h>

